packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/CloudLabelDetector.java

```diff
@@ -55,7 +55,20 @@ public void onFailure(@NonNull Exception e) {
 
   private FirebaseVisionCloudDetectorOptions parseOptions(Map<String, Object> optionsData) {
     final int maxResults = (int) optionsData.get("maxResults");
-    final int modelType = (int) optionsData.get("modelType");
+    final String modelTypeStr = (String) optionsData.get("modelType");
```

Comment:
I made a few changes with how to handle type so no changes to the API will break this.